assets/queries/ansible/aws/rds_auto_minor_version_upgrade_disabled/test/negative.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: negative - create minimal aurora instance in default VPC and default subnet group
+  community.aws.rds_instance:
+    engine: aurora
+    db_instance_identifier: ansible-test-aurora-db-instance
+    instance_type: db.t2.small
+    password: "{{ password }}"
+    username: "{{ username }}"
+    cluster_id: ansible-test-cluster
+    auto_minor_version_upgrade: true
+- name: negative - Create a DB instance using the default AWS KMS encryption key
+  community.aws.rds_instance:
+    id: test-encrypted-db
+    state: present
+    engine: mariadb
+    storage_encrypted: True
+    db_instance_class: db.t2.medium
+    username: "{{ username }}"
+    password: "{{ password }}"
+    allocated_storage: "{{ allocated_storage }}"
+    auto_minor_version_upgrade: yes
+- name: negative - Create a DB instance using the default AWS KMS encryption key
+  community.aws.rds_instance:
+    id: test-encrypted-db
+    state: present
+    engine: mariadb
+    storage_encrypted: True
+    db_instance_class: db.t2.medium
+    username: "{{ username }}"
+    password: "{{ password }}"
+    allocated_storage: "{{ allocated_storage }}"
+    auto_minor_version_upgrade: true
+- name: negative - Basic mysql provisioning example01
+  community.aws.rds:
+    command: create
+    instance_name: new-database
+    db_engine: MySQL
+    size: 10
+    instance_type: db.m1.small
+    username: mysql_admin
+    password: 1nsecure
+    auto_minor_version_upgrade: true
+    tags:
+      Environment: testing
+      Application: cms
```

Comment:
I didn't find the attribute `auto_minor_version_upgrade` in this resource's docs, could you tell me where did you see present? https://docs.ansible.com/ansible/latest/collections/community/aws/rds_module.html#ansible-collections-community-aws-rds-module

Reply:
My bad. I'll remove this sample